site/docs/reference/Connectors/materialization-connectors/Elasticsearch.md

```diff
@@ -0,0 +1,96 @@
+---
+sidebar_position: 2
+---
+
+# Elasticsearch
+
+This connector materializes Flow collections into indices in an Elasticsearch cluster.
+
+[`ghcr.io/estuary/materialize-elasticsearch:dev`](https://ghcr.io/estuary/materialize-elasticsearch:dev) provides the latest connector image. You can also follow the link in your browser to see past image versions.
+
+## Prerequisites
+
+To use this connector, you'll need:
+
+* An Elastic cluster with a known [endpoint](https://www.elastic.co/guide/en/elasticsearch/reference/current/getting-started.html#send-requests-to-elasticsearch)
+  * If the cluster is on the Elastic Cloud, you'll need an Elastic user with a role that grants all privileges on indices in the cluster.
+    See Elastic's documentation on [defining roles](https://www.elastic.co/guide/en/elasticsearch/reference/current/defining-roles.html#roles-indices-priv) and
+    [security privileges](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-privileges.html#privileges-list-indices).
+* At least one Flow collection
+
+:::tip
+If you haven't yet captured your data from its external source, start at the beginning of the [guide to create a dataflow](../../../guides/create-dataflow.md). You'll be referred back to this connector-specific documentation at the appropriate steps.
+:::
+
+
+## Configuration
+
+To use this connector, begin with data in one or more Flow collections.
+Use the below properties to configure an Elasticsearch materialization, which will direct the contents of these Flow collections into Elasticsearch indices.
+
+You must indicate the desired Elasticsearch field type for each field in your Flow collection.
```

Comment:
This is optional: Since Elasticsearch itself is a JSON storage, by default the connector saves the JSON document by translating its types to Elasticsearch as best as it can. `field_overrides` gives users the ability to override the type translation for a specific field (there are more than a single type in Elasticsearch for each JSON data type, for example a string in JSON can be translated to either `text` or `keyword` or both in Elasticsearch, hence the `field_overrides` which lets the user customise the translation).